features/src/test/scala/com/salesforce/op/utils/spark/OPVectorMetadataTest.scala

```diff
@@ -72,24 +76,51 @@ class OPVectorMetadataTest extends PropSpec with TestCommon with PropertyChecks
   )
   val arrVecColTupleGen: Gen[Array[OpVectorColumnTuple]] = Gen.containerOf[Array, OpVectorColumnTuple](vecColTupleGen)
 
+  val sensitiveGen: Gen[SensitiveTuple] = for {
```

Comment:
Nice - I like the generators!

Maybe call this one `sensitiveNameGen` since it's specifically making info for name fields?